modules/os2forms_forloeb/src/Controller/MaestroNotificationController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Drupal\os2forms_forloeb\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Url;
+use Drupal\os2forms_digital_post\Model\Document;
```

Comment:
dependency problem: os2forms_digital_post is listed as dependency in composer.json, and it not required via info.yml file

Reply:
No, it's just suggested (cf. https://github.com/OS2Forms/os2forms/pull/56/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R78-R80).

This is a temporary solution mentioned in the pull request comments.